core/utils.class.inc.php

```diff
@@ -551,7 +551,7 @@ public static function JSONPrettyPrint($json)
 	 * @return false|string
 	 * @throws \Exception
 	 */
-	function Exec($sCmd)
+	public static function Exec($sCmd)
```

Comment:
Do you have examples of this function usages?

Reply:
Ask to Erwan

Reply:
Discussed during PR review.